src/referrals/referrals.controller.ts

```
@@ -62,6 +62,20 @@ export class ReferralsController {
     );
   }
 
+  @UseFilters(new AllExceptionsFilter(APIID.REFERRAL_GET))
+  @Get(':id')
```

ESLint

[error] Replace `':id'` with `":id"`

(prettier/prettier)